CONTRIBUTING.md

```diff
@@ -73,7 +73,7 @@ For workflow diagrams, see the [diagrams](./docs/workflows/) under the `docs` fo
 #### Code structure
 
 - `actions` - Provides specific logic for `trestle-bot` tasks that are packaged as Actions. See [README.md](./actions/README.md) for more information.
-- `entrypoints` - Provides top level logic for specific user-facing tasks. These tasks are not necessarily related in any way so they are not organized into a hierarchical command structure, but they do inherit logic and flags from a base class.
+- `cli` - Provides top level logic for specific user-facing tasks. These tasks are not necessarily related so they are not organized into a hierarchical command structure, but they do share some common modules.
```

Comment:
Should we document the `cli/commands` and `cli/options`? The structure explanation may be helpful to emphasize the cli as the main entrypoint. This could help with documenting distinction of commands and their frequently used options. Let me know what you guys think @gvauter @qduanmu

Reply:
I think we could add some very high level documentation.  I'm hesitant to go into too much detail as it can become difficult to keep the docs up-to-date/in sync with the code.

Reply:
I agree, making too many detailed descriptions will be more maintenance than necessary. I can propose those changes today.